tasks/test_syntax.js

```diff
@@ -77,6 +77,9 @@ function assertSrcContents() {
                     else if(lastPart === 'innerHTML') {
                         logs.push(file + ' : contains .innerHTML (IE failure in SVG)');
                     }
+                    else if(lastPart === 'parentElement') {
+                        logs.push(file + ' : contains .parentElement (IE failure)');
```

Comment:
For reference purposes: https://stackoverflow.com/questions/8685739/difference-between-dom-parentnode-and-parentelement